cdap-validation/bin/ambari.py

```diff
@@ -0,0 +1,87 @@
+import urllib2
+import json
+import re
+import glob, os
+import test_helpers as helpers
+
+# Start Ambari configuration retrieval process
+def ambari_commands(host, subdir, base, cluster_info):
```

Comment:
Which `ambari_commands` are these? That's a very non-descriptive name. Make sure your method names reflect their function.